homeassistant/components/netatmo.py

```diff
@@ -50,7 +64,10 @@ def setup(hass, config):
             "Please check your settings for NatAtmo API.")
         return False
 
-    for component in 'camera', 'sensor':
+    components = [component for component in ('camera', 'sensor')
```

Comment:
That last one. That's how for example the Wink component does it. Do we have camera's: fire discovery to load camera's. 

Another fix would be to update the dependency to not raise an exception.